### ISSUE TITLE
Reaped yard manor - during "salt minion test.ping" - RAET Only

### ISSUE THREAD

**Issue by turtletraction** [saltstack/salt#27269]
Hello,

After the great assistance in the solution of #27229 I have hit a new issue with the test.ping "reaped yard manor". I have a RAET only environment.

Here is the trace and versions-report, again sorry for the malformat due to an old browser.

```
salt minion test.ping -l trace
[DEBUG   ] Reading configuration from /etc/salt/master
[DEBUG   ] Using cached minion ID from /etc/salt/minion_id: minion
[DEBUG   ] Missing configuration file: /root/.saltrc
[TRACE   ] None of the required configuration sections, 'logstash_udp_handler' and 'logstash_zmq_handler', were found the in the configuration. Not loading the Logstash logging handlers module.
[TRACE   ] The required configuration section, 'fluent_handler', was not found the in the configuration. Not loading the fluent logging handlers module.
[DEBUG   ] Configuration file path: /etc/salt/master
[DEBUG   ] Reading configuration from /etc/salt/master
[DEBUG   ] Using cached minion ID from /etc/salt/minion_id: minion
[DEBUG   ] Missing configuration file: /root/.saltrc
[DEBUG   ] RAETEvent Using Jobber Stack at = /var/run/salt/master/master.event520411284146298d1d.uxd

Error sending to '/var/run/salt/master/master.manor.uxd' from '/var/run/salt/master/master.client520411284a205832c4.uxd: [Errno 2] No such file or directory
Reaped yard manor
```

```
Salt Version:
           Salt: 2015.8.0

Dependency Versions:
         Jinja2: 2.8
       M2Crypto: Not Installed
           Mako: Not Installed
         PyYAML: 3.11
          PyZMQ: Not Installed
         Python: 2.7.10 (default, Sep 18 2015, 10:49:43)
           RAET: 0.6.1
        Tornado: 4.2.1
            ZMQ: Not Installed
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: 1.3.9
        libnacl: 1.4.3
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: Not Installed
         pygit2: Not Installed
   python-gnupg: Not Installed
          smmap: Not Installed
        timelib: Not Installed

System Versions:
           dist: redhat 6.4 Santiago
        machine: x86_64
        release: 2.6.32-358.11.1.el6.x86_64
         system: Red Hat Enterprise Linux Server 6.4 Santiago
```
The only reference I can find is #13966 but here the test.ping call never return.

Many thanks in advance for you inputs, work arounds are welcome.

**Comment by turtletraction** [comment]
Here is the directory listing:
```
ls -ltr /var/run/salt/master/master.client520411284a205832c4.uxd
srwxr-x--- 1 root root 0 Sep 21 14:32 /var/run/salt/master/master.client520411284a205832c4.uxd
```
The file exists but is empty in fact all files in this directory are empty but maybe that is by design?

**Comment by jfindlay** [comment]
@turtletraction, thanks for the report.

**Comment by DmitryKuzmenko** [comment]
@turtletraction thank you for the report.

I've investigated your issue. The message you're getting means ```salt``` client you're running can't connect to master. ```salt``` cmdline connects to master via uxd sockets this means you can execute ```salt``` command on the only master host. Also the user executing ```salt``` command should be the same user executing salt master.

Could you please confirm you're doing things in accordance with the above? If so, could you please additionally specify what patches you've applied to salt, because I definitely can see that ```salt minion test.ping``` raises exception on 2015.8.0 because of the inconsistency in ```RAETEvent.get_event()``` interface fixed in 5d4c90c4 that is included in the only 2015.8.1 release.

Status: either won't fix or can't reproduce. @turtletraction waiting for your response.

**Comment by cachedout** [comment]
Because we have not had a response here as requested, I am closing this for the time being. If a response comes back, we can definietely re-open it.
